contracts/lazy_mint/ERC1155/ILazyMintERC1155.sol

```diff
@@ -66,6 +66,8 @@ interface ILazyMintERC1155 {
      */
     struct ClaimConditions {
         uint256 nextConditionIndex;
+        uint256 totalConditionsTillDate;
```

Comment:
`totalConditionsTillDate` naming is pretty interesting haha

Reply:
i wonder if we can name it as `id`? 
since id is a more generic name we can use to uniquely identify the claim condition